qiskit/providers/ibmq/managed/managedjobset.py

```diff
@@ -159,8 +160,8 @@ def retrieve_jobs(self, provider: AccountProvider, refresh: bool = False) -> Non
         matched = pattern.match(first_job.name())
         if not matched:
             raise IBMQJobManagerInvalidStateError(
-                "Job {} is tagged for the job set {} but does not have a proper job name.".format(
-                    first_job.job_id(), self.job_set_id()))
+                'Job {} is tagged for the job set {} but it does not '
```

Comment:
you'll need a comma before the `but` since you're connecting two sentences. Alternatively you can just use the old sentence (i.e. leaving out the `it` after the `but`, making it a dependent clause). Same for line 177.